



<h2 id="cell-section">Cell-based assays</h2>
Cell-based assays
<div class="container grid">
<div class="g-col-3 toolbox" style="text-align:center;">
<div class="tool-container">
<div class="tool-wrapper">
<div class="tool-tooltip-text">Cell-based assays</div>
<div id="cell" class="tool">

<img class="tool-icon" height="100%" src="/toolbox/img/tb_cell-based-assays.png"/></div>
</div>
</div>
</div>
<div class="g-col-9">

<h4>CPE reduction assay</h4>

This antiviral assay measures the ability of a tested compound to protect infected cells from virus-induced cytopathic effects (mostly virus-induced cell death). The cell viability can be determined in a colorimetric assay using MTS/PMS method. 

<h4>Virus yield assay</h4>

This antiviral assay measures the ability of a tested compound to inhibit the viral replication in an infected cell culture through quantifying the viral RNA and the infectious virus loads released in the supernatant. This assay can be used for validation of compounds that show efficacy in CPE-reduction assays and to confirm that the compounds have direct effect on viral replication (not only cytoprotective).

<h4>Delay of treatment assay (Time-of-addition)</h4>

This a test to determine the stage(s) of the viral replication cycle at which the compound exerts its antiviral effect. Briefly, cells are treated with the compound at different time points prior or after the infection with the virus then cells are incubated for example until the end of one or 2 replication cycles. At the end of incubation, the intracellular and extracellular viral RNA loads at each time point are quantified by qRT-PCR to determine at which stage(s) the compound is still effective and when it loses its antiviral efficacy.

<h4>Plaque assay</h4>

This assay is a standard method to quantify infectious virus titers in a sample. Briefly, the test sample is serially diluted then aliquot of each dilution is added to a well in a cell culture plate with incubation to allow virus attachment followed by removal of the viral input and adding overlay of agarose or similar viscous material that limit the spread of viral infection. Under this condition, the attached virion will start to replicate producing more virions that will kill surrounding cells to generate a hole or plaque in the cell monolayer. By removal of the overlay and staining the live cells for e.g. by Crystal violet or methylene blue, this will provide a dark background for clear visualization of plaques. By counting the plaques at certain dilution(s), the plaque forming units (PFU)/mL can be calculated.


<div id="table-cell" class="table">table-cell</div>

```{ojs}
//| output: none
//| echo: false

html`
<table>
${ virus_table("test") }
</table>
`
```

<div class="col-xs-12" style="height:30px;"></div>
</div>
<div class="g-col-1" style="text-align:center;">
</div>
</div>
<div class="col-xs-12" style="height:30px;"></div>



<h2 id="hts-section">High-throughput screening (HTS)</h2>
High-throughput screening (HTS)
<div class="container grid">
<div class="g-col-3 toolbox" style="text-align:center;">
<div class="tool-container">
<div class="tool-wrapper">
<div class="tool-tooltip-text">High-throughput screening (HTS)</div>
<div id="hts" class="tool">

<img class="tool-icon" height="100%" src="/toolbox/img/tb_high-throughput-screening.png"/></div>
</div>
</div>
</div>
<div class="g-col-9">

<h4>High throughput screening (HTS)</h4>

HTS is an approach in antiviral drug discovery involving the use of automated equipment to rapidly test huge libraries of chemical or biological materials with the aim of identifying hit molecules against specific viruses. HTS is usually done in culture plates with 384 or 1536 wells format using high performance liquid/plate handling devices.


<div id="table-hts" class="table">table-hts</div>

```{ojs}
//| output: none
//| echo: false

html`
<table>
${ virus_table("test") }
</table>
`
```

<div class="col-xs-12" style="height:30px;"></div>
</div>
<div class="g-col-1" style="text-align:center;">
</div>
</div>
<div class="col-xs-12" style="height:30px;"></div>



<h2 id="recombinant-section">Recombinant viruses</h2>
Recombinant viruses
<div class="container grid">
<div class="g-col-3 toolbox" style="text-align:center;">
<div class="tool-container">
<div class="tool-wrapper">
<div class="tool-tooltip-text">Recombinant viruses</div>
<div id="recombinant" class="tool">

<img class="tool-icon" height="100%" src="/toolbox/img/tb_recombinant-viruses.png"/></div>
</div>
</div>
</div>
<div class="g-col-9">

<h4>Reverse genetic system</h4>

A reverse genetic system is mostly a plasmid-based system that can be used to insert the desired mutations in the viral genome to study the effect of these mutations on viral replication, virulence, sensitivity to antiviral compounds in vitro and in vivo. 

<h4>Reporter viruses </h4>

Reporter viruses are recombinant viruses engineered to carry reporter genes such as the one coding for fluorescent or luminescent proteins for development of high throughput antiviral assays or easily detection method for in vivo replication.


<div id="table-recombinant" class="table">table-recombinant</div>

```{ojs}
//| output: none
//| echo: false

html`
<table>
${ virus_table("test") }
</table>
`
```

<div class="col-xs-12" style="height:30px;"></div>
</div>
<div class="g-col-1" style="text-align:center;">
</div>
</div>
<div class="col-xs-12" style="height:30px;"></div>



<h2 id="ex-vivo-section">Ex-vivo models</h2>
Ex-vivo models
<div class="container grid">
<div class="g-col-3 toolbox" style="text-align:center;">
<div class="tool-container">
<div class="tool-wrapper">
<div class="tool-tooltip-text">Ex-vivo models</div>
<div id="ex-vivo" class="tool">

<img class="tool-icon" height="75%" src="/toolbox/img/tb_ex-vivo-models.png"/><img class="tool-icon" height="75%" src="/toolbox/img/tb_ex-vivo-models2.png"/></div>
</div>
</div>
</div>
<div class="g-col-9">

<h4>Primary cell culture</h4>

These cultures present several advantages over cell lines, including a better representation of the cellular heterogeneity of tissues, a more faithful transcriptomic and proteomic profile (especially when cultured in 3D) and more realistic functional responses, including drug responses.

<h4>Organoids</h4>

An organoid is a 3D tissue that is mostly derived from stem cells (pluripotent, fetal or adult), and which mimics the key functional, structural and biological complexity of a specified organ. Human organoids are useful tools to study viral infection and efficacy of potential antiviral compounds in a relevant model without the need for more complex animal studies. One example is Human Airway Epithelial (HAE) Cell Cultures that are widely used to study respiratory viral infections.


<div id="table-ex-vivo" class="table">table-ex-vivo</div>

```{ojs}
//| output: none
//| echo: false

html`
<table>
${ virus_table("test") }
</table>
`
```

<div class="col-xs-12" style="height:30px;"></div>
</div>
<div class="g-col-1" style="text-align:center;">
</div>
</div>
<div class="col-xs-12" style="height:30px;"></div>



<h2 id="animal-section">Animal infection models</h2>
Animal infection models
<div class="container grid">
<div class="g-col-3 toolbox" style="text-align:center;">
<div class="tool-container">
<div class="tool-wrapper">
<div class="tool-tooltip-text">Animal infection models</div>
<div id="animal" class="tool">

<img class="tool-icon" height="75%" src="/toolbox/img/tb_animal-infection-models.png"/><img class="tool-icon" height="75%" src="/toolbox/img/tb_animal-infection-models-2.png"/></div>
</div>
</div>
</div>
<div class="g-col-9">

<h4>Rodent models</h4>

Rodent models such as mice, rats and hamsters have been widely used to study viral infections. Such models allow to study host response and pathogenicity for wide variety of viruses. Moreover, these models are beneficial to assess the preclinical efficacy and safety of potential antivirals identified in cell culture assays.

<h4>Zebrafish models</h4>

Zebrafish (Danio rerio) are optically-transparent tropical freshwater fish of the family Cyprinidae that are widely used as vertebrate models of disease. They have remarkable genetic, physiologic and pharmacologic similarities to humans. Compared to rodents, the maintenance and husbandry costs are very low. Interestingly, zebrafish larvae have been shown to be susceptible to infection with some human viruses (herpes simplex virus type 1, influenza A virus, and chikungunya virus, human norovirus). Therefore, such a model can be used to study the pathogenesis as well as test antiviral drugs for specific human viruses in higher capacity and less cost compared to rodent models.


<div id="table-animal" class="table">table-animal</div>

```{ojs}
//| output: none
//| echo: false

html`
<table>
${ virus_table("test") }
</table>
`
```

<div class="col-xs-12" style="height:30px;"></div>
</div>
<div class="g-col-1" style="text-align:center;">
</div>
</div>
<div class="col-xs-12" style="height:30px;"></div>



<h2 id="target-section">Target-based assays</h2>
Target-based assays
<div class="container grid">
<div class="g-col-3 toolbox" style="text-align:center;">
<div class="tool-container">
<div class="tool-wrapper">
<div class="tool-tooltip-text">Target-based assays</div>
<div id="target" class="tool">

<img class="tool-icon" height="100%" src="/toolbox/img/tb_target-based-assays.png"/></div>
</div>
</div>
</div>
<div class="g-col-9">

<h4>Protein, enzyme or molecular mechanism-based assays</h4>


<div id="table-target" class="table">table-target</div>

```{ojs}
//| output: none
//| echo: false

html`
<table>
${ virus_table("test") }
</table>
`
```

<div class="col-xs-12" style="height:30px;"></div>
</div>
<div class="g-col-1" style="text-align:center;">
</div>
</div>
<div class="col-xs-12" style="height:30px;"></div>


